lib/BUILD

```diff
@@ -125,6 +129,8 @@ cc_library(
         "io.h",
         "io_file.h",
         "matrix.h",
+        "mps_simulator.h",
+        "mps_statespace.h",
```

Comment:
qsimh (the hybrid Schrodinger-Feynman simulator) is largely unused at this point. Is it necessary to include these here?

Reply:
Oh my mistake. let me remove it then :) Thank you @95-martin-orion